configs/Ec2UserdataUtils.psm1

```diff
@@ -1627,7 +1627,7 @@ function Get-EventlogOsTemplate {
 function Configure-NxLog {
   param (
     [string] $version = 'c0fe5210f89a', # latest: 'tip'
```

Comment:
git: `072a37a3dae79f9be4e296895365fa37a3eba0f7`

Reply:
I got `a2ec06e2faeb909e645b68db98650de507dd480b` for this one. Your revision looks a couple of months older than this one (maybe a copy/paste error, since this is the same as the one from your first comment?).

Reply:
Yes.